sos/policies/package_managers/flatpak.py

```diff
@@ -15,16 +15,20 @@ class FlatpakPackageManager(PackageManager):
     """Package Manager for Flatpak distributions
     """
 
-    query_command = 'flatpak list --columns=name,version'
+    query_command = 'flatpak list --columns=name,version,branch'
     query_path_command = ''
     files_command = ''
     verify_command = ''
     verify_filter = ''
 
     def _parse_pkg_list(self, pkg_list):
-        for line in pkg_list.splitlines():
-            pkg = line.split()
-            name, version = pkg[0], pkg[1]
-            yield (name, version, None)
+        for line in pkg_list.splitlines()[1:]:
```

Comment:
Why the `[1:]`? The `flatpak list` does not start with a header, the very first line already contains a package - at least in my case..

Reply:
It does in my case, but if I recall correctly, the output is different when called from sos, and there's no header there. I've fixed this in the next version.